test/factories/interview_questions.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :interview_question do
+    id 1
+    interview_id 1
+    question_id 200
+    comment { Faker::Lorem.characters(250) }
```

Comment:
and more magic number

```id 1``` id is populated automatically so no need to set it explicitly
``` interview_id 1```, ```question_id``` investigate how to work with relations with FacrotyGirl and rework